po/de.po

```diff
@@ -81,25 +82,25 @@ msgstr "Installiere Paket %s"
 #: ../src/yumex/__init__.py:815
 #, python-format
 msgid "%s was installed successfully"
-msgstr ""
+msgstr "%s wurde erfolgreich installiert"
 
 #: ../src/yumex/__init__.py:820 ../src/yumex/old_init.py:259
 #, python-format
 msgid "Removing package: %s"
-msgstr "Entferne Paket %s"
+msgstr "Entferne Paket: %s"
 
 #: ../src/yumex/__init__.py:821
 #, python-format
 msgid "%s was removed successfully"
-msgstr ""
+msgstr "%s wurde erfolgreich entfernt"
 
 #: ../src/yumex/__init__.py:826 ../src/yumex/old_init.py:264
 msgid "Updating all available updates"
-msgstr ""
+msgstr "Update alle verfügbaren Aktualisierungen"
```

Comment:
Better: „Installiere alle verfügbaren Aktualisierungen“
Anyway, the original message isn't perfect either. But since it is the "old_init.py" file which will be removed eventually, there is no need to change this imho.